frontend/src/app/[locale]/formPrototype/page.tsx

```diff
@@ -0,0 +1,78 @@
+import { Metadata } from "next";
+import TopLevelError from "src/app/[locale]/error/page";
+import NotFound from "src/app/[locale]/not-found";
+import { ApiRequestError, parseErrorStatus } from "src/errors";
+import withFeatureFlag from "src/hoc/withFeatureFlag";
+import { getFormDetails } from "src/services/fetch/fetchers/formFetcher";
+import { formDetail } from "src/types/formResponseTypes";
+import { WithFeatureFlagProps } from "src/types/uiTypes";
+
+import { redirect } from "next/navigation";
+import { GridContainer } from "@trussworks/react-uswds";
+
+import ApplyForm from "src/components/applyForm/ApplyForm";
+import {
+  validateFormSchema,
+  validateUiSchema,
+} from "src/components/applyForm/validate";
+import BetaAlert from "src/components/BetaAlert";
+
+export function generateMetadata() {
+  const meta: Metadata = {
+    title: `Form test for form React JSON Schema`,
+    description: "this is the form",
+  };
+  return meta;
+}
+
+async function FormPage() {
+  let formData = {} as formDetail;
+  const id = "anyform";
+  try {
+    const response = await getFormDetails(id);
+    formData = response.data;
+  } catch (error) {
+    if (parseErrorStatus(error as ApiRequestError) === 404) {
+      return <NotFound />;
+    }
+    throw error;
+  }
+
+  const { form_json_schema, form_ui_schema } = formData;
+  try {
+    validateUiSchema(form_ui_schema);
+    validateFormSchema(form_json_schema);
+  } catch (error) {
+    console.error("Error validating", error);
+    return <TopLevelError />;
```

Comment:
[non-blocking] eventually we should add an error page for the route

also, I think if you just rethrow the error here it will bubble up to the top level error page fwiw?